zerver/lib/email_mirror.py

```diff
@@ -403,11 +403,14 @@ def is_forwarded(subject: str) -> bool:
 
 def process_stream_message(to: str, message: EmailMessage) -> None:
     subject_header = message.get("Subject", "")
-    subject = strip_from_subject(subject_header) or "(no topic)"
+    subject = strip_from_subject(subject_header) or "(incoming email: no subject)"
```

Comment:
the actual value is to be tweaked, maybe just `(no subject)` etc.

also, do we want to make it just be this value as long as the subject ends up stripped to empty, or do we want two cases?
1. if the realm allows empty topic, then keep this `(no topic)`
2. this new fallback default otherwise

Reply:
Probably those two cases is correct.

Reply:
Tweaked this to `email: no subject` for now, since @alya also suggested dropping the `incoming` and made https://chat.zulip.org/#narrow/stream/101-design/topic/Topic.20name.20.20for.20emails.20without.20subject.20in.20the.20email.20gateway for a proper discussion

Reply:
This should also get a comment, since `(no topic)` would seem like a very valid value to be using here.

Reply:
added a comment